### PR TITLE
Remove deprecated API for watching VPPAPI events

### PR DESCRIPTION
**Pull request by NikitaSkrynnik**
## Comment
https://github.com/FDio/govpp/blob/master/api/api.go#L142

### CODE REVIEW

**Comment by glazychev-art** [comment]
I guess we should use the same approach for https://github.com/networkservicemesh/sdk-vpp/blob/main/pkg/networkservice/up/common.go

File: pkg/networkservice/mechanisms/memif/memifrxmode/metadata.go [hunk 1]
```diff
@@ -2,6 +2,8 @@
 //
 // Copyright (c) 2021-2022 Doc.ai and/or its affiliates.
 //
+// Copyright (c) 2024 Cisco and/or its affiliates.
```
Comment (glazychev-art) [inline pkg/networkservice/mechanisms/memif/memifrxmode/metadata.go:5]: unnecessary change

**Review by glazychev-art — APPROVED** [review]

File: go.mod [hunk 2]
```diff
@@ -10,10 +10,9 @@ require (
 	github.com/google/uuid v1.3.1
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/networkservicemesh/api v1.11.1-0.20231031152153-730abd666985
-	github.com/networkservicemesh/govpp v0.0.0-20240130152300-43babf54855b
-	github.com/networkservicemesh/sdk v0.5.1-0.20240212105149-b1a3e26da857
-	github.com/networkservicemesh/sdk-kernel v0.0.0-20240212105448-f9f740175836
-	github.com/networkservicemesh/vpphelper v0.0.0-20230901145133-a14aecebd1cb
+	github.com/networkservicemesh/govpp v0.0.0-20240115135805-9d7177f85ad9
+	github.com/networkservicemesh/sdk v0.5.1-0.20240117140958-90f0c79a1560
+	github.com/networkservicemesh/sdk-kernel v0.0.0-20240117141240-f32d767554a6
```
Comment (glazychev-art) [inline go.mod:15]: dependencies are downgraded.
Please rebase to the latest main

**Review by glazychev-art — APPROVED** [review]

**Review by denis-tingaikin — APPROVED** [review]
